drivers/ethernet/eth_cyclonev.c

```diff
@@ -315,6 +315,7 @@ static int eth_cyclonev_set_config(const struct device *dev, enum ethernet_confi
 				   const struct ethernet_config *config)
 {
 	struct eth_cyclonev_priv *p = dev->data;
+	const struct eth_cyclonev_config *cv_config = dev->data;
```

Comment:
This looks wrong. It should probably be:
```suggestion
	const struct eth_cyclonev_config *cv_config = dev->config;
```